src/object/schedcontext.c

```diff
@@ -353,10 +353,8 @@ time_t schedContext_updateConsumed(sched_context_t *sc)
 {
     ticks_t consumed = sc->scConsumed;
     if (consumed >= getMaxTicksToUs()) {
-        sc->scConsumed -= getMaxTicksToUs();
```

Comment:
Although this commit message states what the change does, and the implications of the change, It's not clear from the commit message *why* this change is being made. 

 The initial semantics were that when consumed is updated, it's cleared. The use cases I had in mind where that if you are using this info to monitor scheduling of multiple components, then that component needs to be the timeout handler of all components to reason correctly about scheduling, and thus gets all the information. For the examples I wrote this was sufficient.

What is the motivating example here?

Reply:
I'm not sure if this is realistic, but the example I had in mind is where the fault first occurs in a shared resource server with a fault handler that only attempts to recover that server without modifying the donated SC (such as via some alternate recovery SC). In this case it could reply to the client that donated the SC, and the client's fault handler would receive the fault and still be given the appropriate consumed time if it was performing its own accounting.